client/influxdb_test.go

```diff
@@ -547,3 +547,14 @@ func TestClient_NoTimeout(t *testing.T) {
 		t.Fatalf("unexpected error.  expected %v, actual %v", nil, err)
 	}
 }
+
+func TestClient_ParseConnectionStringi_Ipv6(t *testing.T) {
```

Comment:
Also, Go style dictates that this test should be named `TestClient_ParseConnectionString_IPv6`.